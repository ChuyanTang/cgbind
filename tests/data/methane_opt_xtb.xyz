5

C    0.01080    -0.00270   0.00090  
H    -0.57340   -0.81810   -0.45110 
H    -0.45300   0.96810    -0.28230 
H    -0.04020   -0.09890   1.09250  
H    1.05570    -0.04850   -0.36000 
