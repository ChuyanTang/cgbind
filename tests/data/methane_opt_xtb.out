      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. MÃ¤rz, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/07/04 at 18:17:45.925     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/tom/.local/bin/xtb methane_opt_xtb.xyz --chrg 0 --opt
          coordinate file            : methane_opt_xtb.xyz
          omp threads                :                     1
          number of atoms            :                     5
          number of electrons        :                     8
          charge                     :                     0
          spin                       :                   0.0
          first test random number   :      0.30403257014359

########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   8          :
          :  # atomic orbitals                   8          :
          :  # shells                            6          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -4.2209032 -0.422090E+01  0.250E+00   16.60       0.0  T
   2     -4.2347468 -0.138436E-01  0.914E-01   16.42       1.0  T
   3     -4.2348799 -0.133123E-03  0.510E-01   16.32       1.0  T
   4     -4.2349317 -0.518102E-04  0.102E-01   16.20       1.0  T
   5     -4.2349317 -0.908692E-08  0.494E-03   16.20       9.1  T
   6     -4.2349318 -0.608701E-07  0.513E-04   16.20      87.2  T
   7     -4.2349318 -0.130973E-09  0.142E-04   16.20     314.6  T

   *** convergence criteria satisfied after 7 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.5781190             -15.7314
         2        2.0000           -0.4676026             -12.7241
         3        2.0000           -0.4652978             -12.6614
         4        2.0000           -0.4645022             -12.6397 (HOMO)
         5                          0.1306573               3.5554 (LUMO)
         6                          0.1841489               5.0109
         7                          0.1967070               5.3527
         8                          0.2077847               5.6541
      -------------------------------------------------------------
                  HL-Gap            0.5951595 Eh           16.1951 eV
             Fermi-level           -0.1669225 Eh           -4.5422 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.000 sec (  6.337%)
 Dispersion                     ...        0 min,  0.000 sec (  1.584%)
 integral evaluation            ...        0 min,  0.000 sec ( 13.077%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  1.235%)
 iterations                     ...        0 min,  0.000 sec ( 51.477%)
 molecular gradient             ...        0 min,  0.000 sec ( 19.817%)
 printout                       ...        0 min,  0.000 sec (  4.270%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.173807451325 Eh    ::
         :: gradient norm              0.030103660801 Eh/a0 ::
         :: HOMO-LUMO gap             16.195115217650 eV    ::
         ::.................................................::
         :: SCC energy                -4.234931790771 Eh    ::
         :: -> isotropic ES            0.001858175451 Eh    ::
         :: -> anisotropic ES          0.002758248622 Eh    ::
         :: -> anisotropic XC          0.004123603675 Eh    ::
         :: -> dispersion             -0.000664664507 Eh    ::
         :: repulsion energy           0.061124342358 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                 9          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/Î± :
          :   maximium RF displ.         1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from model Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by   0.000000000000000E+000
 Lowest  eigenvalues of input Hessian
   -0.000000   -0.000000   -0.000000    0.000000    0.000000    0.000000
    0.089070    0.090252    0.111716    0.112778    0.115957    0.390279
    1.038334    1.063614    1.084950
 Highest eigenvalues
    0.112778    0.115957    0.390279    1.038334    1.063614    1.084950
 

........................................................................
.............................. CYCLE    1 ..............................
........................................................................
   1     -4.2349318 -0.423493E+01  0.801E-06   16.20       0.0  T
   2     -4.2349318 -0.337508E-13  0.443E-06   16.20   10102.3  T
   3     -4.2349318 -0.177636E-14  0.155E-06   16.20   28870.9  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -4.1738074 Eh     change        0.9277174E-08 Eh
   gradient norm :     0.0301032 Eh/Î±   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.0900301 Î±      lambda       -0.2413257E-02
   maximum displ.:     0.0713847 Î±      in ANC's #6, #5, #2, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................
   1     -4.2457285 -0.424573E+01  0.166E-01   17.17       0.0  T
   2     -4.2457458 -0.173011E-04  0.821E-02   17.18       1.0  T
   3     -4.2457460 -0.209049E-06  0.323E-02   17.18       1.4  T
   4     -4.2457460 -0.226537E-07  0.319E-03   17.19      14.0  T
   5     -4.2457460 -0.237256E-08  0.404E-04   17.19     110.7  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -4.1751845 Eh     change       -0.1377094E-02 Eh
   gradient norm :     0.0054553 Eh/Î±   predicted    -0.1216410E-02 ( -11.67%)
   displ. norm   :     0.0151212 Î±      lambda       -0.7253885E-04
   maximum displ.:     0.0131622 Î±      in ANC's #6, #5, #7, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................
   1     -4.2476831 -0.424768E+01  0.298E-02   17.37       0.0  T
   2     -4.2476838 -0.611323E-06  0.146E-02   17.37       3.1  T
   3     -4.2476838 -0.200426E-07  0.594E-03   17.37       7.5  T
   4     -4.2476838 -0.104905E-08  0.102E-03   17.37      44.0  T
   5     -4.2476838 -0.371707E-09  0.917E-05   17.37     487.8  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -4.1752178 Eh     change       -0.3326395E-04 Eh
   gradient norm :     0.0007418 Eh/Î±   predicted    -0.3627848E-04 (   9.06%)
   displ. norm   :     0.0033072 Î±      lambda       -0.1258582E-05
   maximum displ.:     0.0021065 Î±      in ANC's #5, #6, #1, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................
   1     -4.2473851 -0.424739E+01  0.718E-03   17.35       0.0  T
   2     -4.2473851 -0.145413E-07  0.403E-03   17.35      11.1  T
   3     -4.2473851 -0.516201E-09  0.100E-03   17.35      44.5  T
   4     -4.2473851 -0.132783E-10  0.160E-04   17.35     278.7  T
   5     -4.2473851 -0.905231E-11  0.162E-05   17.35    2759.8  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -4.1752184 Eh     change       -0.6211728E-06 Eh
   gradient norm :     0.0002309 Eh/Î±   predicted    -0.6288220E-06 (   1.23%)
   displ. norm   :     0.0010825 Î±      lambda       -0.1399139E-06
   maximum displ.:     0.0007726 Î±      in ANC's #5, #6, #1, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 4 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -0.0014110 Eh       -0.8854 kcal/mol
 total RMSD          :         0.0440520 a0        0.0233 Ã
 total power (kW/mol):        -0.9261253 (step)-1600.8453 (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.002 sec
 optimizer setup                ...        0 min,  0.000 sec (  2.280%)
 model hessian                  ...        0 min,  0.000 sec (  3.359%)
 ANC generation                 ...        0 min,  0.000 sec (  3.575%)
 coordinate transformation      ...        0 min,  0.000 sec (  0.781%)
 single point calculation       ...        0 min,  0.002 sec ( 74.227%)
 optimization log               ...        0 min,  0.000 sec (  3.790%)
 hessian update                 ...        0 min,  0.000 sec (  0.514%)
 rational function              ...        0 min,  0.000 sec (  5.342%)

================
 final structure:
================
5
 xtb: 6.2.3 (830e466)
C        -0.00014290339042   -0.00004901075612    0.00004126691178
H        -0.56442251606130   -0.80664066634041   -0.44991725996754
H        -0.43635319130215    0.94866701930821   -0.28478552125473
H        -0.02462157966166   -0.09967344454670    1.07750449345486
H         1.02544019041554   -0.04240389766498   -0.34284297914437
 
 Bond Distances (Angstroems)
 ---------------------------
C1-H2=1.0823         C1-H3=1.0823         C1-H4=1.0823         C1-H5=1.0822         H2-C1=1.0823         H3-C1=1.0823         
H4-C1=1.0823         H5-C1=1.0822         
 C  H  Rav=1.0823 sigma=0.0001  Rmin=1.0822  Rmax=1.0823     4
 
 selected bond angles (degree)
 --------------------
H3-C1-H2=109.49                H4-C1-H2=109.48                H4-C1-H3=109.48                H5-C1-H2=109.46                
H5-C1-H3=109.44                H5-C1-H4=109.47                
 
 selected dihedral angles (degree)
 ---------------------------------

           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   8          :
          :  # atomic orbitals                   8          :
          :  # shells                            6          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -4.2473851 -0.424739E+01  0.450E-07   17.35       0.0  T
   2     -4.2473851 -0.355271E-14  0.264E-07   17.35  100000.0  T
   3     -4.2473851  0.266454E-14  0.161E-08   17.35  100000.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.5818103             -15.8319
         2        2.0000           -0.4674677             -12.7204
         3        2.0000           -0.4674508             -12.7200
         4        2.0000           -0.4674229             -12.7192 (HOMO)
         5                          0.1699973               4.6259 (LUMO)
         6                          0.2289385               6.2297
         7                          0.2290162               6.2318
         8                          0.2291043               6.2342
      -------------------------------------------------------------
                  HL-Gap            0.6374202 Eh           17.3451 eV
             Fermi-level           -0.1487128 Eh           -4.0467 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.000 sec
 SCC setup                      ...        0 min,  0.000 sec ( 12.156%)
 Dispersion                     ...        0 min,  0.000 sec (  1.091%)
 integral evaluation            ...        0 min,  0.000 sec ( 21.818%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  1.558%)
 iterations                     ...        0 min,  0.000 sec ( 20.052%)
 molecular gradient             ...        0 min,  0.000 sec ( 32.727%)
 printout                       ...        0 min,  0.000 sec (  6.494%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.175218421325 Eh    ::
         :: gradient norm              0.000230878889 Eh/a0 ::
         :: HOMO-LUMO gap             17.345086144098 eV    ::
         ::.................................................::
         :: SCC energy                -4.247385119261 Eh    ::
         :: -> isotropic ES            0.002034396362 Eh    ::
         :: -> anisotropic ES          0.002324446032 Eh    ::
         :: -> anisotropic XC          0.003588661123 Eh    ::
         :: -> dispersion             -0.000661035504 Eh    ::
         :: repulsion energy           0.072166692342 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.5818103             -15.8319
         2        2.0000           -0.4674677             -12.7204
         3        2.0000           -0.4674508             -12.7200
         4        2.0000           -0.4674229             -12.7192 (HOMO)
         5                          0.1699973               4.6259 (LUMO)
         6                          0.2289385               6.2297
         7                          0.2290162               6.2318
         8                          0.2291043               6.2342
      -------------------------------------------------------------
                  HL-Gap            0.6374202 Eh           17.3451 eV
             Fermi-level           -0.1487128 Eh           -4.0467 eV

     #   Z        covCN         q      C6AA      Î±(0)
     1   6 C      3.702    -0.156    23.686     6.959
     2   1 H      0.925     0.039     2.463     2.454
     3   1 H      0.925     0.039     2.463     2.454
     4   1 H      0.925     0.039     2.463     2.454
     5   1 H      0.925     0.039     2.464     2.454

 Mol. C6AA /auÂ·bohrâ¶  :        123.838214
 Mol. C8AA /auÂ·bohrâ¸  :       2297.173628
 Mol. Î±(0) /au        :         16.776907


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  C   3.994        H    3 0.998    H    2 0.998    H    4 0.998    H    5 0.998
     2  H   0.998        C    1 0.998
     3  H   0.998        C    1 0.998
     4  H   0.998        C    1 0.998
     5  H   0.998        C    1 0.998

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.000      -0.000      -0.000
   full:        0.000       0.000      -0.000       0.000
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:       -0.000       0.000       0.000       0.000      -0.000       0.000
  q+dip:       -0.001       0.001       0.001       0.000      -0.000       0.000
   full:       -0.001       0.000       0.001       0.000      -0.000       0.000

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       16.0424989
   center of mass at/Ã    :       -0.0001043      -0.0000399       0.0000283
  moments of inertia/uÂ·ÃÂ² :        0.3147910E+01   0.3148284E+01   0.3149355E+01
rotational constants/cmâ»Â¹ :        0.5355182E+01   0.5354547E+01   0.5352725E+01

 * 4 selected distances

     #   Z        #   Z                                     value/Ã
     1   6 C      2   1 H                                 1.0823420
     1   6 C      3   1 H                                 1.0823437 (max)
     1   6 C      4   1 H                                 1.0823360
     1   6 C      5   1 H                                 1.0822127 (min)

 * 1 distinct bonds (by element types)

   Z      Z             #   av. dist./Ã        max./Ã        min./Ã
   1 H    6 C           4     1.0823086     1.0823437     1.0822127


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY               -4.175218421325 Eh   |
          | GRADIENT NORM               0.000230878889 Eh/Î± |
          | HOMO-LUMO GAP              17.345086144098 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/07/04 at 18:17:45.936     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.010 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.010 sec
 * ratio c/w:     0.997 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.001 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.001 sec
 * ratio c/w:     0.999 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.003 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.003 sec
 * ratio c/w:     1.000 speedup

